channels/ae.m3u

```diff
@@ -77,7 +77,7 @@ https://jmc-live.ercdn.net/hawastvhd/hawastvhd.m3u8
 https://jmc-live.ercdn.net/hawastvhd/hawastvhd_720p.m3u8
 #EXTINF:-1 tvg-id="KhyberMiddleEastTV.ae" tvg-name="Khyber Middle East TV" tvg-country="AE" tvg-language="" tvg-logo="" group-title="",Khyber Middle East TV
 https://dcunilive83-lh.akamaihd.net/i/dclive_1@16122/master.m3u8
-#EXTINF:-1 tvg-id="KhyberNewsTV.ae" tvg-name="Khyber News TV" tvg-country="AE" tvg-language="" tvg-logo="" group-title="",Khyber News TV
+#EXTINF:-1 tvg-id="KhyberNewsTV.ae" tvg-name="Khyber News TV" tvg-country="AE" tvg-language="" tvg-logo="" group-title="Sports",Khyber News TV
```

Comment:
KhyberNews is a news channel, not Sports channel. In fact, it must be in Pakistani playlist and the language is Pashto
Scroll down the home page of
https://khybernews.tv/